packages/legend-graph/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_ValueSpecificationTransformer.ts

```diff
@@ -469,3 +476,20 @@ export function V1_transformLambdaFunction(
   lambda.body = V1_transformLambdaBody(lambdaFunc, false);
   return lambda;
 }
+
+export function V1_transformValueSpecification(
```

Comment:
This kind of change worries me a beat :sob: because you create this function which is supposed to be generic, but you don't remove the exporting of `class V1_ValueSpecificationTransformer`, so we end up with 2 ways to transform in the codebase. could we stop exporting the class? Also, it seems like you haven't change other places that use `new ValueSpecificationTransformer` in this class itself, is there a reason for that?